src/test/ui-fulldeps/extern-mod-syntax.rs

```diff
@@ -3,8 +3,8 @@
 #![allow(unused_imports)]
 #![feature(rustc_private)]
 
-extern crate rustc_serialize;
-use rustc_serialize::json::Object;
+extern crate libc;
+use libc::c_void;
```

Comment:
Needed a replacement here. `c_void` is available in libc on all platforms I think. Not sure though.